files/es/learn/common_questions/set_up_a_local_testing_server/index.html

```diff
@@ -36,70 +36,87 @@ <h2 id="Archivos_locales_versus_archivos_remotos">Archivos locales versus archiv
+<p>Sabrás que estás ejecutando el ejemplo desde un archivo local porque la dirección web comenzará con <code>file://</code>, seguido de la ruta al archivo en tu disco duro local. Por el contrario, si ves uno de nuestros ejemplos alojados en GitHub (o un ejemplo en algún otro servidor remoto), la dirección web tendrá <code>http://</code> o <code>https://</code> al principio, para mostrar que el archivo ha sido recibido a través de HTTP.</p>
 
 <h2 id="El_problema_de_probar_archivos_locales">El problema de probar archivos locales</h2>
 
-<p>Algunos ejemplos no se ejecutarán si los abre como archivos locales. Esto puede deberse a una variedad de razones, siendo las más probables:</p>
+<p>Algunos ejemplos no se ejecutarán si se los abre como archivos locales. Esto puede deberse a una variedad de razones, siendo las más probables:</p>
 
 <ul>
- <li><strong>Cuentan con peticiones asincrónicas</strong>. Algunos navegadores (incluido chrome) no ejecutarán solicitudes asíncronas (consulte <a href="/en-US/docs/Learn/JavaScript/Client-side_web_APIs/Fetching_data">Obtención de datos desde el servidor</a>) si acaba de ejecutar el ejemplo desde un archivo local. Esto se debe a las restricciones de seguridad (para obtener más información sobre la seguridad web, lee <a href="/en-US/docs/Learn/Server-side/First_steps/Website_security">La seguridad del sitio web</a>).</li>
- <li><strong>Cuenta con un lenguaje de servidor</strong>. Los lenguajes de servidor (como PHP o Python) requieren de un servidor especial para interpretar el código y entregar los resultados.</li>
+ <li><strong>Cuentan con peticiones asincrónicas</strong>. Algunos navegadores (incluido chrome) no ejecutarán solicitudes asíncronas (consulte <a href="/en-US/docs/Learn/JavaScript/Client-side_web_APIs/Fetching_data">Obtención de datos desde el servidor</a>) si se ejecuta el ejemplo desde un archivo local. Esto se debe a restricciones de seguridad (para obtener más información sobre la seguridad web, lee <a href="/es/docs/Learn/Server-side/First_steps/Website_security">La seguridad del sitio web</a>).</li>
+ <li><strong>Cuentan con un lenguaje de servidor</strong>. Los lenguajes de servidor (como PHP o Python) requieren de un servidor especial para interpretar el código y entregar los resultados.</li>
+ <li><strong>Incluyen otros archivos</strong>. Los navegadores comunmente tratan a los pedidos para cargar recursos formulados utilizando el esquema <code>file://</code>, como solicitudes de recursos de origenes cruzados (<em>cross-origin requests</em>). Por ello, si se carga un archivo local que incluye otros archivos locales, puede dispararse un error <a href="/en-US/docs/Glossary/CORS">CORS</a>.</li>
```

Comment:
URL a su versión en español
```suggestion
 <li><strong>Incluyen otros archivos</strong>. Los navegadores comunmente tratan a los pedidos para cargar recursos formulados utilizando el esquema <code>file://</code>, como solicitudes de recursos de origenes cruzados (<em>cross-origin requests</em>). Por ello, si se carga un archivo local que incluye otros archivos locales, puede dispararse un error <a href="/es/docs/Glossary/CORS">CORS</a>.</li>
```